webapp/src/ts/modules/reports/reports-content.component.html

```diff
@@ -35,11 +35,11 @@
           </div>
 
           <div *ngIf="summaries[$index]?.subject?.type" class="subject">
-            <a class="name" *ngIf="summaries[$index]?.subject._id || summaries[$index].fields?.patient_uuid"
-              [routerLink]="['/contacts', summaries[$index].subject._id || summaries[$index].fields.patient_uuid]"
+            <a class="name" *ngIf="summaries[$index]?.subject._id || summaries[$index].patient._id"
```

Comment:
Cool. 
I think we need to nullcheck patient here, in case it's not set: 
```suggestion
            <a class="name" *ngIf="summaries[$index]?.subject._id || summaries[$index]?.patient?._id"
```
And same goes for the other span.